by-language/java-qa/src/test/java/io/crate/qa/JdbcTest.java

```diff
@@ -0,0 +1,118 @@
+package stock_jdbc;
+
+import java.sql.DriverManager;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import io.crate.testing.CrateTestCluster;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@RunWith(JUnit4.class)
+public class JdbcTest {
+
+    @ClassRule
+    public static final CrateTestCluster TEST_CLUSTER = CrateTestCluster
+        .fromURL("https://cdn.crate.io/downloads/releases/nightly/crate-latest.tar.gz")
+        .settings(Map.of("psql.port", 55433))
+        .build();
+    public static final String URL = "jdbc:postgresql://localhost:55433/doc?user=crate";
```

Comment:
It's not strictly required to use `crate-testing` here, because CI/GHA would also provide a running CrateDB instance.

But well, it also tests `crate-testing`, which isn't a total bad thing. The clear drawback is that it currently only works on Linux. If you think we should level up here, let's take it as chance to track it appropriately.

Reply:
For example, we could look into whether "Testcontainers for Java" could be used here on behalf of another iteration, to at least give developers on non-Linux platforms a better experience.

Reply:
I agree to use testcontainers for this one, but can be done as a followup.